dataflow/src/backlog/mod.rs

```diff
@@ -231,6 +241,37 @@ impl WriteHandle {
     pub(crate) fn is_partial(&self) -> bool {
         self.partial
     }
+
+    /// Evict `count` randomly selected keys from state and return them along with the number of
+    /// bytes that will be freed once the underlying `evmap` applies the operation.
+    pub fn evict_random_key(&mut self, rng: &mut ThreadRng) -> u64 {
+        let mut bytes_to_be_freed = 0;
+        if self.mem_size > 0 {
+            match self.handle.empty_at_index(rng.gen()) {
```

Comment:
Shouldn't this generate something that's in the range of indices? Or is it automatically wrapped around so it always hits a bucket?

Reply:
I believe rahashmap automatically wraps. @fintelia ?